packages/search/src/elasticsearch/dbhelper.ts

```diff
@@ -57,7 +63,10 @@ export const updateComposition = async (id: string, body: ICompositionBody) => {
   return response
 }
 
-export const searchForDuplicates = async (body: IBirthCompositionBody) => {
+export const searchForDuplicates = async (
```

Comment:
Should we remove this function altogether now as it's in the `deduplicate/service.ts`?